x/examples/fetch-psiphon/main.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2023 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"path"
+	"time"
+
+	"github.com/Jigsaw-Code/outline-sdk/x/psiphon"
+)
+
+var debugLog log.Logger = *log.New(io.Discard, "", 0)
+
+func init() {
+	flag.Usage = func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage: %s [flags...] <url>\n", path.Base(os.Args[0]))
+		flag.PrintDefaults()
+	}
+}
+
+func main() {
+	verboseFlag := flag.Bool("v", false, "Enable debug output")
+	configFlag := flag.String("config", "", "Config file")
```

Comment:
Suggest to be more specific about the config format.

```suggestion
	configFlag := flag.String("config", "", "A Psiphon config file")
```